src/Nest/Analysis/Plugins/Icu/Collation/IcuCollationAlternate.cs

```diff
@@ -0,0 +1,17 @@
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Nest
+{
+	/// <summary>
+	/// Sets the alternate handling for strength quaternary to be either shifted or non-ignorable.
+	/// Which boils down to ignoring punctuation and whitespace.
+	/// </summary>
+	[JsonConverter(typeof(StringEnumConverter))]
+	public enum IcuCollationAlternate
+	{
+		[EnumMember(Value="shifted")] Shifted,
+		[EnumMember(Value="non-ignorable")] NonIgnorable
```

Comment:
The ICU analysis plugin has some inconsistencies like this:

https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu-collation.html

https://github.com/elastic/elasticsearch/blob/master/plugins/analysis-icu/src/main/java/org/elasticsearch/index/analysis/IcuCollationTokenFilterFactory.java#L127

I opened a couple issues on elasticsearch for this but not yet for this option will do so now.